bin/test_travis.sh

```diff
@@ -11,6 +11,7 @@ python setup.py install --symengine-dir=$our_install_dir
 # Test python wrappers
 if [[ "${WITH_SAGE}" != "yes" ]]; then
     nosetests -v
+    py.test -v symengine/tests/test_*.py  # pytest gets confused if we install first
```

Comment:
See my reply to your above comment. Regarding `$PYTHON_SOURCE_DIR`, do you mean like this:
`py.test -v $PYTHON_SOURCE_DIR/symengine/tests/test_*.py`